### PR TITLE
feat(env): adding an env variable to override size format

### PR DESCRIPTION
**Pull request by MartinFillon**
The env variable is EZA_SIZE_STYLE as proposed in eza-community/eza#657

### CODE REVIEW

**Comment by cfxegbert** [comment]
Why is this needed?  Wouldn't a command alias be sufficient?

File: src/options/view.rs [hunk 1]
```diff
@@ -302,12 +302,25 @@ impl SizeFormat {
...
+            },
+            _ => None,
+        };
+
         let flag = matches.has_where(|f| f.matches(&flags::BINARY) || f.matches(&flags::BYTES))?;
 
-        Ok(match flag {
-            Some(f) if f.matches(&flags::BINARY) => Self::BinaryBytes,
-            Some(f) if f.matches(&flags::BYTES) => Self::JustBytes,
+        Ok(match (env_var, flag) {
```
Comment (cfxegbert) [inline src/options/view.rs:318]: Command line should take priority over the environment variable.

File: src/options/vars.rs [hunk 2]
```diff
@@ -64,6 +64,8 @@ pub static EZA_MIN_LUMINANCE: &str = "EZA_MIN_LUMINANCE";
 /// Any explicit use of `--icons=WHEN` overrides this behavior.
 pub static EZA_ICONS_AUTO: &str = "EZA_ICONS_AUTO";
 
+pub static EZA_SIZE_STYLE: &str = "EZA_SIZE_STYLE";
```
Comment (cfxegbert) [inline src/options/vars.rs:67]: EZA_SIZE_FORMAT would be a better name.  Event in the code you use size_format.

**Review by cfxegbert — COMMENTED** [review]
Documentation would also need to be updated.

File: src/options/view.rs (hunk 1 above)
Comment (MartinFillon) [reply src/options/view.rs:318]: I agree with that BUT, at that point this PR become kinda unuseful. So I might as well close it ?

Comment (MartinFillon) [reply src/options/view.rs:318]: up @cfxegbert

**Pull request closed without merging**